search-scores-and-weights.md

```diff
@@ -16,38 +15,42 @@ and weights to documents based on various parameters, influencing their ranking
 in the search results. Here is our different scores, each serving a unique
 purpose:
 
-1. **Similarity**: Represents the primary signal in our scoring mechanism. It
-   denotes the measurement of how closely a document aligns with the user's
-   query, reflecting the relevance of the document to the search criteria. It is
-   not a static precomputed score but a dynamic metric that is computed on the
-   fly from the search results. This ensures that the relevance is tied to the
+1. [**Similarity**](sql/2023-07-19-modify-score_type-add-similarity.sql):
+   Represents the primary signal in our scoring mechanism. It denotes the
+   measurement of how closely a document aligns with the user's query,
+   reflecting the relevance of the document to the search criteria. It is not a
+   static precomputed score but a dynamic metric that is computed on the fly
+   from the search results. This ensures that the relevance is tied to the
    specific query, going beyond simple keyword matching. Documents with higher
    similarity scores are considered more relevant. By prioritizing similarity as
    the first signal in our scoring process, we aim to deliver search results
    that are more accurate.
 
-1. **Recency**: This score considers the temporal aspect of documents,
-   prioritizing recently added or updated content. A document's recency is
-   crucial in reflecting the latest information available to users.
+1. [**Recency**](sql/schema2.sql.public_new.sql): This score considers the
+   temporal aspect of documents, prioritizing recently added or updated content.
+   A document's recency is crucial in reflecting the latest information
+   available to users.
 
-1. **Traffic**: The frequency with which users consult a document influences its
-   score. Popular or frequently accessed documents are given higher scores with
-   the help of web traffic logs, indicating their relevance and importance to
-   users. Warning: The home page is rated really high since it's where every
-   user land at first.
+1. [**Traffic**](sql/compute-traffic-score.sql): The frequency with which users
+   consult a document influences its score. Popular or frequently accessed
+   documents are given higher scores with the help of web traffic logs,
+   indicating their relevance and importance to users. Warning: The home page is
+   rated really high since it's where every user land at first.
 
-1. **Current**: This score determines whether a document is currently accessible
```

Comment:
for each score, I think it would be useful to explain both end of the scale.

  * 0.0: oldest document
  * 1.0: most recent document

Reply:
So it would be for recency as current is only either 1 or 0. I will add a scale for each score

Reply:
this is done